boxsdk/object/chunked_uploader.py

```diff
@@ -0,0 +1,45 @@
+# pylint: disable=line-too-long
```

Comment:
This shouldn't be disabled here; we should fix whatever is causing this and address the lint error.